### ISSUE TITLE
Gift cards admin general stats

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#2400]
*Initially suggested by @piamancini*

## User story 

As an organization admin, I can view general statistics about my generated gift cards to know how the campaign is going.

## Best solution for this problem

Above the gift cards list on `/collective/edit/gift-cards`, display:
- Total number generated + total value
- Number claimed with amount spent
- Number expired with amount spent
- Number not claimed yet with total value

![image](https://user-images.githubusercontent.com/1556356/64324966-9793d400-cfc7-11e9-91a6-88b60040dd44.png)

**Comment by piamancini** [comment]
nice! I had forgotten about this thanks Ben

**Comment by alanna** [comment]
This would be great. It would be even better if it showed this per batch created. For example OSC creates gift cards for various reasons (OSCA events, sponsoring conferences, etc) and it would be really useful to see these stats. A workaround if that's too complicated would be to provide a CSV download of all the data together with the batch name as a column so admins could figure out the stats per batch themselves.

**Comment by Betree** [comment]
@alanna Good point, we don't have a notion of "batch" at the moment but we should definitely create it asap to make this kind of features possible in the future. I'll work on an issue to spec that out.
